client/src/modules/login/login.html

```diff
@@ -3,15 +3,26 @@
     <div class="row">
       <div class="col-md-4 col-md-offset-4 col-xs-12">
 
-        <div ng-if="!LoginCtrl.installationExist" class="text-center" style="margin-top : calc(50vh - 190px); max-width: 400px; margin-right: auto; margin-left: auto;">
+        <div 
+          ng-if="!LoginCtrl.installationExist && LoginCtrl.finishInstallationChecking && !LoginCtrl.installationCheckingError" 
+          lass="text-center" style="margin-top : calc(50vh - 190px); max-width: 400px; margin-right: auto; margin-left: auto;">
```

Comment:
It looks like this has broken the "class" attribute.  It now reads `lass="..."`.